src/components/Tool/FixSendNewsletter/GetInvalidNewsletter.graphql

```diff
@@ -0,0 +1,36 @@
+query GetInvalidNewsletter($accountListId: ID!) {
+  contacts(
+    accountListId: $accountListId
+    contactsFilter: { newsletter: NO_VALUE }
```

Comment:
Same deceased filter needed eventually

Reply:
I'll make a note to add it later to prevent any errors for now